determined-ai/determined
# chore: migrate `react-router-dom` from v5 to v6

## Description

[DET-7851]



## Test Plan
- Check all pages if router works as expected
- Check code if there's something I miss



## Commentary (optional)

I followed [this instruction](https://github.com/remix-run/react-router/discussions/8753) and [this](https://reactrouter.com/docs/en/v6/upgrading/v5)




## Checklist

See [Release Note](../docs/release-notes/README.md) for details.





[DET-7851]: https://determinedai.atlassian.net/browse/DET-7851?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
